package.json

```diff
@@ -51,9 +51,9 @@
     "@lunie/cosmos-api": "git+https://git@github.com/hicommonwealth/cosmos-api.git#develop",
     "@lunie/cosmos-keys": "^0.0.11",
     "@openzeppelin/contracts": "^2.4.0",
-    "@polkadot/api": "1.12.0-beta.12",
+    "@polkadot/api": "^1.14.1",
```

Comment:
We should specify specific API versions rather than using ^, because we don't want to accidentally grab a greater version than expected and break the app.